jsonnetfile.lock.json

```diff
@@ -26,7 +26,7 @@
       "name": "grafana-builder",
       "source": {
         "git": {
-          "remote": "https://github.com/grafana/jsonnet-libs",
+          "remote": "https://github.com/kausalco/public",
```

Comment:
unwanted change?

Reply:
That's what I got when did `make generate-in-docker` with latest container image :man_shrugging:

Reply:
The repo was renamed at some point, not sure why this is happening in this direction though. It's definitely a no-op though, so I'm ok with moving forward on green CI.